portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,7 +50,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
   throws IOException {
     String name = request.getParameter("user-name");
     String comment = request.getParameter("user-comment");
+    
+    User user = new User(name, comment))
```

Comment:
missing semi-colon on this line

Reply:
+1 on this, we can't build your code because it doesn't compile :(